docs/source/_static/css/custom.css

```diff
@@ -0,0 +1,5 @@
+img.screenshot {
+    /* Copied from div.admonition */
+    box-shadow: 0 .2rem .5rem var(--pst-color-shadow),0 0 .0625rem var(--pst-color-shadow);
+    border-color: var(--pst-color-info);
+}
```

Comment:
Couldn't this CSS rule include `max-width` and `max-height` properties to limit the size of the images, rather than compressing the images themselves? Doesn't necessarily have to set the `max-{width,height}` properties, any CSS implementation is fine. I just don't want to rely on compressed images; it's too easy for contributors to upload a screenshot from their 4K display to the docs without compressing it first.

Reply:
The problem with that is that our screenshots are of varying dimensions. Some are of a main area with a side-panel widget (in landscape orientation) and some are of just the side-panel (typically in portrait). The bothersome part wasn't that the screen shots were too tall or too wide, but rather that the text in the screen shots was larger than the text in the documentation. I'm not sure if that problem can be solved with CSS alone.